docs/modules/mssql.md

```diff
@@ -3,15 +3,15 @@
 |![](https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/Warning.svg/50px-Warning.svg.png) | This module is EXPERIMENTAL. It is under heavy development and remains subject to the possibility of breaking changes. |
 |---|---|
 
-Open Service Broker for Azure contains three Azure SQL Database services. These services enable you to select the most appropriate provision scenario for your needs. These services are:
+Open Service Broker for Azure (OSBA) contains three Azure SQL Database services. These services enable you to select the most appropriate provision scenario for your needs. These services are:
 
 | Service Name | Description |
 |--------------|-------------|
 | `azure-sql` | Provision both a SQL Server DBMS and a database. |
 | `azure-sql-dbms-only` | Provision only a SQL Server Database Management System (DBMS). This can be used to provision multiple databases at a later time. |
-| `azure-sql-db-only` | Provision a new database only upon a previously provisioned DBMS. |
+| `azure-sql-database-only` | Provision a new database only upon a previously provisioned DBMS. |
 
-The `azure-sql` service allows you to provision both a DBMS and a database. This service is ready to use upon successful provisioning. You can not provision additional databases onto an instance provisioned through this service. The `azure-sql-dbms-only` and `azure-sql-db-only` services, on the other hand, can be combined to provision multiple databases on a single DBMS.  For more information on each service, refer to the descriptions below.
+The `azure-sql` service allows you to provision both a DBMS and a database. This service is ready to use upon successful provisioning. You can not provision additional databases onto an instance provisioned through this service. The `azure-sql-dbms-only` and `azure-sql-database-only` services, on the other hand, can be combined to provision multiple databases on a single DBMS.  For more information on each service, refer to the descriptions below.
```

Comment:
s/`This service is ready to use upon successful provisioning`/`When the provision operation is successful, the database will be ready to use`